apps/theming-designer/src/components/ThemingDesigner.tsx

```diff
@@ -0,0 +1,202 @@
+import * as React from 'react';
+import { FabricPalette } from './FabricPalette';
+import { AccessibilityChecker } from './AccessibilityChecker';
+import { SemanticSlots } from './SemanticSlots';
+import { Header } from './Header';
+import { Samples } from './Samples';
+import { mergeStyles } from '@uifabric/merge-styles';
+import {
+  Stack,
+  loadTheme,
+  ITheme,
+  getTheme,
+  BaseComponent,
+  IColor,
+  getColorFromString,
+  IconButton,
+  ThemeProvider,
+  createTheme
+} from '../../../../packages/office-ui-fabric-react/lib/index';
+import { ThemeDesignerColorPicker } from './ThemeDesignerColorPicker';
+
+import { ThemeGenerator, themeRulesStandardCreator, BaseSlots, IThemeRules } from 'office-ui-fabric-react/lib/ThemeGenerator';
+import { isDark } from 'office-ui-fabric-react/lib/utilities/color/shades';
+
+export interface IThemingDesignerState {
+  primaryColor: IColor;
+  textColor: IColor;
+  backgroundColor: IColor;
+  theme?: ITheme;
+  themeRules?: IThemeRules;
+}
+
+const appClassName = mergeStyles({
+  display: 'flex'
+});
+
+const sideBarClassName = mergeStyles({
+  flex: '0 0 400px',
+  background: '#fff'
+});
+
+const mainCardsClassName = mergeStyles({
+  flexDirection: 'column',
+  background: '#fff',
+  flex: 1,
+  margin: 0,
+  padding: 0
+});
+
+export class ThemingDesigner extends BaseComponent<{}, IThemingDesignerState> {
+  constructor(props: any) {
+    super(props);
+
+    const themeRules = themeRulesStandardCreator();
+    ThemeGenerator.insureSlots(themeRules, isDark(themeRules[BaseSlots[BaseSlots.backgroundColor]].color!));
+
+    this.state = {
+      theme: getTheme(true),
+      themeRules: themeRules,
+      primaryColor: getColorFromString('#ffa500')!,
+      textColor: getColorFromString('#0078d4')!,
+      backgroundColor: getColorFromString('#323130')!
+    };
+
+    this._onPrimaryColorPickerChange = this._onPrimaryColorPickerChange.bind(this);
+    this._onTextColorPickerChange = this._onTextColorPickerChange.bind(this);
+    this._onBkgColorPickerChange = this._onBkgColorPickerChange.bind(this);
+  }
+
+  public componentWillUnmount(): void {
+    // remove temp styles
+    const root = document.querySelector('.samples') as HTMLElement;
+    // const root = document.getElementsByClassName('samples');
+    if (root) {
```

Comment:
If the theming designer is only affecting the samples block instead of the whole page, probably don't need this logic anymore.